website/src/pages/index.js

```diff
@@ -287,7 +287,7 @@ function Home() {
 {
   "name": "synapseml",
   "conf": {
-      "spark.jars.packages": "com.microsoft.azure:synapseml_2.12:0.9.5",
+      "spark.jars.packages": "com.microsoft.azure:synapseml_2.12:0.9.4",
```

Comment:
I think this clarification was already added to website

Reply:
Very true!